provider/terraform/property_override.rb

```diff
@@ -26,6 +26,33 @@ module OverrideFields
       attr_reader :default
       attr_reader :sensitive # Adds `Sensitive: true` to the schema
       attr_reader :validation # Adds a ValidateFunc to the schema
+
+      # ===========
+      # Custom code
+      # ===========
+      # Property Updates are used when a resource is updateable but
+      # resource.input is true.  In this case, only individual
+      # properties can be updated.  The value of this attribute should
+      # be the path to a template which will be compiled. This code is placed
+      # *inline* in the obj := { ... } definition - it is not a custom
+      # function, it is a custom statement.  Note that this cannot
+      # be used for nested properties, as they are not present in the
+      # obj := {...} statement.
+      attr_reader :property_update
```

Comment:
`property_` is redundant now that we are in the context of a property. What about something like `update_statement` or `update_statement_template`?

Are you passing any variables to the template to be compiled? If so, please document here. If not, consider simply inlining the code instead of using a template (unless you see from your experience with Disk that this statement tends to be complex and long).

Reply:
Done.